gui/session/objectives/autociv_statsOverlay.js

```diff
@@ -73,34 +82,34 @@ AutocivControls.StatsOverlay = class
             this.update()
     }
 
-    indexDefeated(state)
+    indexNonDefeated(state)
```

Comment:
this method return a list, the method name doesn't say that

listOfNotDefeated(state)

You can add comment  that it will return player index if or find better name

Reply:
Renamed it to "listOfNotDefeated"